server/src/test/java/org/elasticsearch/search/aggregations/bucket/terms/LongKeyedBucketOrdsTests.java

```diff
@@ -104,7 +104,6 @@ private void collectsFromSingleBucketCase(LongKeyedBucketOrds ords) {
         }
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/58353")
```

Comment:
Is this removed by accident here? I don't see the relation to the change in the rest test above. Also the error  reproduction from #58353 still works for me.